2025-02-02 11:09:45,334 - DEBUG - Starting new HTTPS connection (1): api.telegram.org:443
2025-02-02 11:09:46,164 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getMe HTTP/1.1" 200 269
2025-02-02 11:09:46,165 - INFO - Bot ulandi - @QuizEnglishwords_bot
2025-02-02 11:09:46,467 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getMe HTTP/1.1" 200 269
2025-02-02 11:09:46,469 - DEBUG - Starting new HTTPS connection (1): api.telegram.org:443
2025-02-02 11:09:57,335 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=1&timeout=20 HTTP/1.1" 200 375
2025-02-02 11:09:57,338 - DEBUG - Starting new HTTPS connection (1): api.telegram.org:443
2025-02-02 11:09:58,564 - DEBUG - https://api.telegram.org:443 "POST /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/sendMessage?chat_id=6572793464&text=%F0%9F%8E%AF+%3Cb%3EQuiz+Bot%3C%2Fb%3Ega+xush+kelibsiz%21%0A%0ABot+imkoniyatlari%3A%0A1.+So%27zlarni+bittadan+kiritish%0A2.+Ko%27p+so%27zlarni+bir+vaqtda+kiritish%0A3.+Fayl+%28.txt%2C+.xlsx%2C+.docx%2C+.pdf%29+yuklash%0A4.+Quiz+o%27yini+%2810%2C+20%2C+30+ta+savol%29%0A5.+Har+bir+savolga+10+soniya+vaqt%0A%0ABoshlash+uchun+quyidagi+tugmalardan+birini+tanlang+%F0%9F%91%87&reply_markup=%7B%22keyboard%22%3A+%5B%5B%7B%22text%22%3A+%22%5Cud83d%5Cudcdd+So%27zlarni+kiritish%22%7D%2C+%7B%22text%22%3A+%22%5Cud83d%5Cudcdd+Ko%27p+so%27z+kiritish%22%7D%5D%2C+%5B%7B%22text%22%3A+%22%5Cud83d%5Cudcc1+File+yuklash%22%7D%5D%5D%2C+%22resize_keyboard%22%3A+true%7D&parse_mode=HTML HTTP/1.1" 200 626
2025-02-02 11:09:58,564 - INFO - Yangi foydalanuvchi: 6572793464
2025-02-02 11:10:03,885 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=788610203&timeout=20 HTTP/1.1" 200 342
2025-02-02 11:10:04,415 - DEBUG - https://api.telegram.org:443 "POST /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/sendMessage?chat_id=6572793464&text=So%27zlarni+quyidagi+formatda+kiriting%3A%0A%0Aapple+-+olma%0Abook+-+kitob%0Acat+-+mushuk%0Adog+-+it%0A%0AHar+bir+so%27z+yangi+qatordan+yozilishi+kerak%21%0AQuiz%27ni+boshlash+uchun+%27quiz%27+so%27zini+yuboring.&parse_mode=HTML HTTP/1.1" 200 448
2025-02-02 11:10:07,064 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=788610204&timeout=20 HTTP/1.1" 200 342
2025-02-02 11:10:07,067 - DEBUG - Starting new HTTPS connection (1): api.telegram.org:443
2025-02-02 11:10:08,810 - DEBUG - https://api.telegram.org:443 "POST /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/sendMessage?chat_id=6572793464&text=%E2%9D%8C+1+ta+so%27z+noto%27g%27ri+formatda%0A%0AYana+so%27z+qo%27shish+uchun+xuddi+shunday+formatda+yuboring.%0AQuiz%27ni+boshlash+uchun+%27quiz%27+so%27zini+yuboring.&parse_mode=HTML HTTP/1.1" 200 405
2025-02-02 11:10:08,810 - INFO - Foydalanuvchi 6572793464: 0 ta so'z qo'shildi
2025-02-02 11:10:18,938 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=788610205&timeout=20 HTTP/1.1" 200 315
2025-02-02 11:10:19,249 - DEBUG - https://api.telegram.org:443 "POST /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/sendMessage?chat_id=6572793464&text=%E2%9D%8C+Quiz+boshlash+uchun+kamida+4+ta+so%27z+kiritilgan+bo%27lishi+kerak%21&parse_mode=HTML HTTP/1.1" 200 330
2025-02-02 11:10:26,325 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=788610206&timeout=20 HTTP/1.1" 200 336
2025-02-02 11:10:45,072 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=788610207&timeout=20 HTTP/1.1" 200 886
2025-02-02 11:10:57,450 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=788610208&timeout=20 HTTP/1.1" 200 336
2025-02-02 11:11:17,626 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=788610209&timeout=20 HTTP/1.1" 200 23
2025-02-02 11:11:38,069 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=788610209&timeout=20 HTTP/1.1" 200 23
2025-02-02 11:11:58,487 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=788610209&timeout=20 HTTP/1.1" 200 23
2025-02-02 11:12:18,754 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=788610209&timeout=20 HTTP/1.1" 200 23
2025-02-02 11:12:39,127 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=788610209&timeout=20 HTTP/1.1" 200 23
2025-02-02 11:12:59,303 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=788610209&timeout=20 HTTP/1.1" 200 23
2025-02-02 11:13:19,574 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=788610209&timeout=20 HTTP/1.1" 200 23
2025-02-02 11:13:39,962 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=788610209&timeout=20 HTTP/1.1" 200 23
2025-02-02 11:14:00,122 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=788610209&timeout=20 HTTP/1.1" 200 23
2025-02-02 11:14:20,401 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=788610209&timeout=20 HTTP/1.1" 200 23
2025-02-02 11:14:40,569 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=788610209&timeout=20 HTTP/1.1" 200 23
2025-02-02 11:15:00,951 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=788610209&timeout=20 HTTP/1.1" 200 23
2025-02-02 11:15:21,134 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=788610209&timeout=20 HTTP/1.1" 200 23
2025-02-02 11:15:41,277 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=788610209&timeout=20 HTTP/1.1" 200 23
2025-02-02 11:15:52,436 - DEBUG - Starting new HTTPS connection (1): api.telegram.org:443
2025-02-02 11:15:53,479 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getMe HTTP/1.1" 200 269
2025-02-02 11:15:53,481 - INFO - Bot ulandi - @QuizEnglishwords_bot
2025-02-02 11:15:53,812 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getMe HTTP/1.1" 200 269
2025-02-02 11:15:53,815 - DEBUG - Starting new HTTPS connection (1): api.telegram.org:443
2025-02-02 11:15:59,936 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=1&timeout=20 HTTP/1.1" 200 375
2025-02-02 11:15:59,940 - DEBUG - Starting new HTTPS connection (1): api.telegram.org:443
2025-02-02 11:16:01,676 - DEBUG - https://api.telegram.org:443 "POST /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/sendMessage?chat_id=6572793464&text=%F0%9F%8E%AF+%3Cb%3EQuiz+Bot%3C%2Fb%3Ega+xush+kelibsiz%21%0A%0ABot+imkoniyatlari%3A%0A1.+So%27zlarni+bittadan+kiritish%0A2.+Ko%27p+so%27zlarni+bir+vaqtda+kiritish%0A3.+Fayl+%28.txt%2C+.xlsx%2C+.docx%2C+.pdf%29+yuklash%0A4.+Quiz+o%27yini+%2810%2C+20%2C+30+ta+savol%29%0A5.+Har+bir+savolga+10+soniya+vaqt%0A%0ABoshlash+uchun+quyidagi+tugmalardan+birini+tanlang+%F0%9F%91%87&reply_markup=%7B%22keyboard%22%3A+%5B%5B%7B%22text%22%3A+%22%5Cud83d%5Cudcdd+So%27zlarni+kiritish%22%7D%2C+%7B%22text%22%3A+%22%5Cud83d%5Cudcdd+Ko%27p+so%27z+kiritish%22%7D%5D%2C+%5B%7B%22text%22%3A+%22%5Cud83d%5Cudcc1+File+yuklash%22%7D%5D%5D%2C+%22resize_keyboard%22%3A+true%7D&parse_mode=HTML HTTP/1.1" 200 626
2025-02-02 11:16:01,676 - INFO - Yangi foydalanuvchi: 6572793464
2025-02-02 11:16:04,643 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=788610210&timeout=20 HTTP/1.1" 200 336
2025-02-02 11:16:21,440 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=788610211&timeout=20 HTTP/1.1" 200 1222
2025-02-02 11:16:21,442 - DEBUG - Starting new HTTPS connection (1): api.telegram.org:443
2025-02-02 11:16:22,671 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getFile?file_id=BQACAgIAAxkBAAIBE2efDGRsz73IJMxJCPf93HP2ZMQtAAIbagAC3fT4SNrXM6JpBBeNNgQ HTTP/1.1" 200 195
2025-02-02 11:16:22,941 - DEBUG - https://api.telegram.org:443 "POST /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/sendMessage?chat_id=6572793464&text=%F0%9F%93%A5+Fayl+yuklanmoqda...&parse_mode=HTML HTTP/1.1" 200 292
2025-02-02 11:16:23,603 - DEBUG - https://api.telegram.org:443 "GET /file/bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/documents/file_0.pdf HTTP/1.1" 200 285528
2025-02-02 11:16:27,175 - DEBUG - https://api.telegram.org:443 "POST /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/sendMessage?chat_id=6572793464&text=%E2%9C%85+Fayl+muvaffaqiyatli+yuklandi%21%0A%F0%9F%93%9A+25+ta+so%27z+qo%27shildi.%0A%0AQuiz%27ni+boshlash+uchun+%27quiz%27+so%27zini+yuboring.&parse_mode=HTML HTTP/1.1" 200 383
2025-02-02 11:16:27,694 - DEBUG - https://api.telegram.org:443 "POST /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/sendMessage?chat_id=6572793464&text=%F0%9F%93%9D+Nechta+savol+tuzishni+xohlaysiz%3F&reply_markup=%7B%22keyboard%22%3A+%5B%5B%7B%22text%22%3A+%2210+ta+savol%22%7D%2C+%7B%22text%22%3A+%2220+ta+savol%22%7D%2C+%7B%22text%22%3A+%2230+ta+savol%22%7D%5D%5D%2C+%22resize_keyboard%22%3A+true%7D&parse_mode=HTML HTTP/1.1" 200 305
2025-02-02 11:16:34,345 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=788610212&timeout=20 HTTP/1.1" 200 315
2025-02-02 11:16:40,483 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=788610213&timeout=20 HTTP/1.1" 200 322
2025-02-02 11:16:40,900 - DEBUG - https://api.telegram.org:443 "POST /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/sendMessage?chat_id=6572793464&text=%F0%9F%93%9D+Savol+1%2F10%3A%0A%0A%27sho%E2%80%98ba.++sun%E2%80%99iy+intellekt+ni+harbiy+sohalarda+qo%E2%80%98llash+muammolari.%27+so%27zining+tarjimasi+qaysi%3F%0A%0A%E2%8F%B1+Vaqt%3A+10+soniya&reply_markup=%7B%22keyboard%22%3A+%5B%5B%7B%22text%22%3A+%224%22%7D%2C+%7B%22text%22%3A+%22fan%22%7D%5D%2C+%5B%7B%22text%22%3A+%22kommunikatsiya+texnologiyalari+va+aloqa+harbiy+instituti+boshlig%5Cu2018i%22%7D%2C+%7B%22text%22%3A+%2271-66%2C+ichki+tel%3A+59+-36%2C+59-46.%22%7D%5D%5D%2C+%22resize_keyboard%22%3A+true%7D&parse_mode=HTML HTTP/1.1" 200 424
2025-02-02 11:16:54,012 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=788610214&timeout=20 HTTP/1.1" 200 382
2025-02-02 11:16:54,207 - DEBUG - https://api.telegram.org:443 "POST /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/sendMessage?chat_id=6572793464&text=%E2%8F%B1+Vaqt+tugadi%21&parse_mode=HTML HTTP/1.1" 200 279
2025-02-02 11:16:55,021 - DEBUG - https://api.telegram.org:443 "POST /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/sendMessage?chat_id=6572793464&text=%F0%9F%93%9D+Savol+2%2F10%3A%0A%0A%27konferensiya+ni+o%E2%80%98tkazish+uchun+barcha+shart%27+so%27zining+tarjimasi+qaysi%3F%0A%0A%E2%8F%B1+Vaqt%3A+10+soniya&reply_markup=%7B%22keyboard%22%3A+%5B%5B%7B%22text%22%3A+%225.+%5C%22zamonaviy+texnologiyalar+va+innovatsion+yechimlarning+iqtisodiyot%2C+ilm%22%7D%2C+%7B%22text%22%3A+%22sharoitlar+yaratiladi%3A++kompyuter%2C%22%7D%5D%2C+%5B%7B%22text%22%3A+%22variant+da+yoki++axborot%22%7D%2C+%7B%22text%22%3A+%223%22%7D%5D%5D%2C+%22resize_keyboard%22%3A+true%7D&parse_mode=HTML HTTP/1.1" 200 392
2025-02-02 11:17:07,428 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=788610215&timeout=20 HTTP/1.1" 200 335
2025-02-02 11:17:07,795 - DEBUG - https://api.telegram.org:443 "POST /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/sendMessage?chat_id=6572793464&text=%E2%8F%B1+Vaqt+tugadi%21&parse_mode=HTML HTTP/1.1" 200 279
2025-02-02 11:17:08,244 - DEBUG - https://api.telegram.org:443 "POST /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/sendMessage?chat_id=6572793464&text=%F0%9F%93%9D+Savol+3%2F10%3A%0A%0A%27mudofaa+vazirligi+axborot%27+so%27zining+tarjimasi+qaysi%3F%0A%0A%E2%8F%B1+Vaqt%3A+10+soniya&reply_markup=%7B%22keyboard%22%3A+%5B%5B%7B%22text%22%3A+%22%5Cu0441+%5Cu043f%5Cu043e%5Cu043c%5Cu043e%5Cu0449%5Cu044c%5Cu044e+%5Cu0441%5Cu043f%5Cu0435%5Cu0446%5Cu0438%5Cu0430%5Cu043b%5Cu0438+%5Cu0437%5Cu0438%5Cu0440%5Cu043e%5Cu0432%5Cu0430%5Cu043d%5Cu043d%5Cu043e%5Cu0439+%5Cu0438%5Cu043d%5Cu0444%5Cu043e%5Cu0440%5Cu043c%5Cu0430%5Cu0446%5Cu0438%5Cu043e%5Cu043d%5Cu043d%5Cu043e%22%7D%2C+%7B%22text%22%3A+%22maqolalar+matni+to%5Cu2018liq+3%22%7D%5D%2C+%5B%7B%22text%22%3A+%22kommunikatsiya++texnologiyalari++va+aloqa++harbiy%22%7D%2C+%7B%22text%22%3A+%22keltirilishi+hamda+ular+maqolada+havola+%5B1%5D+qilinishi+lozim.+ilmiy%22%7D%5D%5D%2C+%22resize_keyboard%22%3A+true%7D&parse_mode=HTML HTTP/1.1" 200 368
2025-02-02 11:17:15,197 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=788610216&timeout=20 HTTP/1.1" 200 549
2025-02-02 11:17:15,603 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=788610217&timeout=20 HTTP/1.1" 200 549
2025-02-02 11:17:15,604 - DEBUG - https://api.telegram.org:443 "POST /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/sendMessage?chat_id=6572793464&text=%E2%9D%8C+Noto%27g%27ri%21%0ATo%27g%27ri+javob%3A+kommunikatsiya++texnologiyalari++va+aloqa++harbiy&parse_mode=HTML HTTP/1.1" 200 343
2025-02-02 11:17:15,815 - DEBUG - https://api.telegram.org:443 "POST /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/sendMessage?chat_id=6572793464&text=%F0%9F%93%9D+Savol+4%2F10%3A%0A%0A%27%28a4%27+so%27zining+tarjimasi+qaysi%3F%0A%0A%E2%8F%B1+Vaqt%3A+10+soniya&reply_markup=%7B%22keyboard%22%3A+%5B%5B%7B%22text%22%3A+%224%22%7D%2C+%7B%22text%22%3A+%22%5Cu0432%5Cu043e%5Cu0435%5Cu043d%5Cu043d%5Cu044b%5Cu0439+%5Cu0438%5Cu043d%5Cu0441%5Cu0442%5Cu0438%5Cu0442%5Cu0443%5Cu0442+%5Cu0438%5Cu043d%5Cu0444%5Cu043e%5Cu0440%5Cu043c%5Cu0430%5Cu0446%5Cu0438%5Cu043e%5Cu043d%5Cu043d%5Cu043e%22%7D%5D%2C+%5B%7B%22text%22%3A+%22format+%29%2C+rasm+%28sxema%2C+grafik%29+8x10++sm+dan+oshmasligi+kerak%2C+matn+chegarasi%22%7D%2C+%7B%22text%22%3A+%223%22%7D%5D%5D%2C+%22resize_keyboard%22%3A+true%7D&parse_mode=HTML HTTP/1.1" 200 346
2025-02-02 11:17:15,815 - DEBUG - https://api.telegram.org:443 "POST /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/sendMessage?chat_id=6572793464&text=%E2%9D%8C+Noto%27g%27ri%21%0ATo%27g%27ri+javob%3A+kommunikatsiya++texnologiyalari++va+aloqa++harbiy&parse_mode=HTML HTTP/1.1" 200 343
2025-02-02 11:17:16,221 - DEBUG - https://api.telegram.org:443 "POST /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/sendMessage?chat_id=6572793464&text=%F0%9F%93%9D+Savol+5%2F10%3A%0A%0A%2771-66%2C+ichki+tel%3A+59+-36%2C+59-46.%27+so%27zining+tarjimasi+qaysi%3F%0A%0A%E2%8F%B1+Vaqt%3A+10+soniya&reply_markup=%7B%22keyboard%22%3A+%5B%5B%7B%22text%22%3A+%22ma%5Cu2019lumotlar++uchun++telefonlar+%3A+%2B998+70+202%22%7D%2C+%7B%22text%22%3A+%22%5Cu043a%5Cu043e%5Cu043c%5Cu043c%5Cu0443%5Cu043d%5Cu0438%5Cu043a%5Cu0430%5Cu0446%5Cu0438%5Cu043e%5Cu043d%5Cu043d%5Cu044b%5Cu0445+%5Cu0442%5Cu0435%5Cu0445%5Cu043d%5Cu043e%5Cu043b%5Cu043e%5Cu0433%5Cu0438%5Cu0439+%5Cu0438+%5Cu0441%5Cu0432%5Cu044f%5Cu0437%5Cu0438%22%7D%5D%2C+%5B%7B%22text%22%3A+%22kommunikatsiya++texnologiyalari++va+aloqa++harbiy%22%7D%2C+%7B%22text%22%3A+%22yil+5-fevraliga+qadar+e-xat%3A%22%7D%5D%5D%2C+%22resize_keyboard%22%3A+true%7D&parse_mode=HTML HTTP/1.1" 200 375
2025-02-02 11:17:27,757 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=788610218&timeout=20 HTTP/1.1" 200 360
2025-02-02 11:17:27,898 - DEBUG - https://api.telegram.org:443 "POST /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/sendMessage?chat_id=6572793464&text=%E2%8F%B1+Vaqt+tugadi%21&parse_mode=HTML HTTP/1.1" 200 279
2025-02-02 11:17:28,107 - DEBUG - https://api.telegram.org:443 "POST /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/sendMessage?chat_id=6572793464&text=%F0%9F%93%9D+Savol+6%2F10%3A%0A%0A%273%27+so%27zining+tarjimasi+qaysi%3F%0A%0A%E2%8F%B1+Vaqt%3A+10+soniya&reply_markup=%7B%22keyboard%22%3A+%5B%5B%7B%22text%22%3A+%22ma%5Cu2019lumotlar++uchun++telefonlar+%3A+%2B998+70+202%22%7D%2C+%7B%22text%22%3A+%22sho%5Cu2018ba.+ta%5Cu2019lim+tizimida+raqamli+texnologiyalarni+joriy+etishning+ilmiy+-uslubiy%22%7D%5D%2C+%5B%7B%22text%22%3A+%22respublika++ilmiy%22%7D%2C+%7B%22text%22%3A+%22ilmiy%22%7D%5D%5D%2C+%22resize_keyboard%22%3A+true%7D&parse_mode=HTML HTTP/1.1" 200 344
2025-02-02 11:17:30,061 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=788610219&timeout=20 HTTP/1.1" 200 328
2025-02-02 11:17:30,365 - DEBUG - https://api.telegram.org:443 "POST /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/sendMessage?chat_id=6572793464&text=%E2%9D%8C+Noto%27g%27ri%21%0ATo%27g%27ri+javob%3A+sho%E2%80%98ba.+ta%E2%80%99lim+tizimida+raqamli+texnologiyalarni+joriy+etishning+ilmiy+-uslubiy&parse_mode=HTML HTTP/1.1" 200 383
2025-02-02 11:17:30,565 - DEBUG - https://api.telegram.org:443 "POST /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/sendMessage?chat_id=6572793464&text=%F0%9F%93%9D+Savol+7%2F10%3A%0A%0A%27respublika++ilmiy%27+so%27zining+tarjimasi+qaysi%3F%0A%0A%E2%8F%B1+Vaqt%3A+10+soniya&reply_markup=%7B%22keyboard%22%3A+%5B%5B%7B%22text%22%3A+%222%22%7D%2C+%7B%22text%22%3A+%223%22%7D%5D%2C+%5B%7B%22text%22%3A+%22nazariy+konferensiyasi++tashkiliy++qo%5Cu2018mitasining%22%7D%2C+%7B%22text%22%3A+%22punkti%2C+176%22%7D%5D%5D%2C+%22resize_keyboard%22%3A+true%7D&parse_mode=HTML HTTP/1.1" 200 360
2025-02-02 11:17:35,472 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=788610220&timeout=20 HTTP/1.1" 200 312
2025-02-02 11:17:35,783 - DEBUG - https://api.telegram.org:443 "POST /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/sendMessage?chat_id=6572793464&text=%E2%9D%8C+Noto%27g%27ri%21%0ATo%27g%27ri+javob%3A+nazariy+konferensiyasi++tashkiliy++qo%E2%80%98mitasining&parse_mode=HTML HTTP/1.1" 200 347
2025-02-02 11:17:35,984 - DEBUG - https://api.telegram.org:443 "POST /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/sendMessage?chat_id=6572793464&text=%F0%9F%93%9D+Savol+8%2F10%3A%0A%0A%27%D1%82.%D0%BF.+%D0%BB%D0%B0%D0%BA%D0%BE%D1%86%D0%B5%D0%BD%D0%B8%D0%BD%D0%B0.+%E2%80%93+%D1%80%D0%BE%D1%81%D1%82%D0%BE%D0%B2%27+so%27zining+tarjimasi+qaysi%3F%0A%0A%E2%8F%B1+Vaqt%3A+10+soniya&reply_markup=%7B%22keyboard%22%3A+%5B%5B%7B%22text%22%3A+%22%5Cu043d%5Cu0430-%5Cu0434%5Cu043e%5Cu043d%5Cu0443%3A+%5Cu0438%5Cu0437%5Cu0434%5Cu0430%5Cu0442%5Cu0435%5Cu043b%5Cu044c%5Cu0441%5Cu0442%5Cu0432%5Cu043e+%5Cu00ab%5Cu0443%5Cu0447%5Cu0438%5Cu0442+%5Cu0435%5Cu043b%5Cu044c%5Cu00bb%2C+2011.+%5Cu2013+231+%5Cu0441.%22%7D%2C+%7B%22text%22%3A+%22ilmiy%22%7D%5D%2C+%5B%7B%22text%22%3A+%22variant+da+yoki++axborot%22%7D%2C+%7B%22text%22%3A+%22strategiyalari+%5Cu201d+nomli+respublika+ilmiy%22%7D%5D%5D%2C+%22resize_keyboard%22%3A+true%7D&parse_mode=HTML HTTP/1.1" 200 463
2025-02-02 11:17:43,466 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=788610221&timeout=20 HTTP/1.1" 200 316
2025-02-02 11:17:43,779 - DEBUG - https://api.telegram.org:443 "POST /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/sendMessage?chat_id=6572793464&text=%E2%9D%8C+Noto%27g%27ri%21%0ATo%27g%27ri+javob%3A+%D0%BD%D0%B0-%D0%B4%D0%BE%D0%BD%D1%83%3A+%D0%B8%D0%B7%D0%B4%D0%B0%D1%82%D0%B5%D0%BB%D1%8C%D1%81%D1%82%D0%B2%D0%BE+%C2%AB%D1%83%D1%87%D0%B8%D1%82+%D0%B5%D0%BB%D1%8C%C2%BB%2C+2011.+%E2%80%93+231+%D1%81.&parse_mode=HTML HTTP/1.1" 200 487
2025-02-02 11:17:44,074 - DEBUG - https://api.telegram.org:443 "POST /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/sendMessage?chat_id=6572793464&text=%F0%9F%93%9D+Savol+9%2F10%3A%0A%0A%27fan+integratsiyasi%3A+texnologik+innovatsiyalar+orqali+jamiyatni+rivojlantirish%22%27+so%27zining+tarjimasi+qaysi%3F%0A%0A%E2%8F%B1+Vaqt%3A+10+soniya&reply_markup=%7B%22keyboard%22%3A+%5B%5B%7B%22text%22%3A+%22sho%5Cu2018ba.+ta%5Cu2019lim+tizimida+raqamli+texnologiyalarni+joriy+etishning+ilmiy+-uslubiy%22%7D%2C+%7B%22text%22%3A+%227.+%5C%22it+va+ilm%22%7D%5D%2C+%5B%7B%22text%22%3A+%22sho%5Cu2018ba.+axborot+texnologiyalarining+innovatsion+rivojlanish+strategiyalari+va%22%7D%2C+%7B%22text%22%3A+%22%5Cu043a%5Cu043e%5Cu043c%5Cu043c%5Cu0443%5Cu043d%5Cu0438%5Cu043a%5Cu0430%5Cu0446%5Cu0438%5Cu043e%5Cu043d%5Cu043d%5Cu044b%5Cu0445+%5Cu0442%5Cu0435%5Cu0445%5Cu043d%5Cu043e%5Cu043b%5Cu043e%5Cu0433%5Cu0438%5Cu0439+%5Cu0438+%5Cu0441%5Cu0432%5Cu044f%5Cu0437%5Cu0438%22%7D%5D%5D%2C+%22resize_keyboard%22%3A+true%7D&parse_mode=HTML HTTP/1.1" 200 422
2025-02-02 11:17:58,106 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=788610222&timeout=20 HTTP/1.1" 200 374
2025-02-02 11:17:58,512 - DEBUG - https://api.telegram.org:443 "POST /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/sendMessage?chat_id=6572793464&text=%E2%8F%B1+Vaqt+tugadi%21&parse_mode=HTML HTTP/1.1" 200 279
2025-02-02 11:17:58,816 - DEBUG - https://api.telegram.org:443 "POST /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/sendMessage?chat_id=6572793464&text=%F0%9F%93%9D+Savol+10%2F10%3A%0A%0A%272.+%22ta%E2%80%99lim+tizimida+raqamli+texnologiyalarni+joriy+etishning+ilmiy%27+so%27zining+tarjimasi+qaysi%3F%0A%0A%E2%8F%B1+Vaqt%3A+10+soniya&reply_markup=%7B%22keyboard%22%3A+%5B%5B%7B%22text%22%3A+%225.+%5C%22zamonaviy+texnologiyalar+va+innovatsion+yechimlarning+iqtisodiyot%2C+ilm%22%7D%2C+%7B%22text%22%3A+%22%5Cu0432%5Cu043e%5Cu0435%5Cu043d%5Cu043d%5Cu044b%5Cu0439+%5Cu0438%5Cu043d%5Cu0441%5Cu0442%5Cu0438%5Cu0442%5Cu0443%5Cu0442+%5Cu0438%5Cu043d%5Cu0444%5Cu043e%5Cu0440%5Cu043c%5Cu0430%5Cu0446%5Cu0438%5Cu043e%5Cu043d%5Cu043d%5Cu043e%22%7D%5D%2C+%5B%7B%22text%22%3A+%22uslubiy%22%7D%2C+%7B%22text%22%3A+%221%22%7D%5D%5D%2C+%22resize_keyboard%22%3A+true%7D&parse_mode=HTML HTTP/1.1" 200 416
2025-02-03 11:03:16,910 - DEBUG - Starting new HTTPS connection (1): api.telegram.org:443
2025-02-03 11:03:19,474 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getMe HTTP/1.1" 200 269
2025-02-03 11:03:19,476 - INFO - Bot ulandi - @QuizEnglishwords_bot
2025-02-03 11:03:19,731 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getMe HTTP/1.1" 200 269
2025-02-03 11:03:19,733 - DEBUG - Starting new HTTPS connection (1): api.telegram.org:443
2025-02-03 11:03:24,560 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=1&timeout=20 HTTP/1.1" 409 143
2025-02-03 11:03:24,583 - ERROR - Threaded polling exception: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
2025-02-03 11:03:24,586 - ERROR - Exception traceback:
Traceback (most recent call last):
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 1234, in __threaded_polling
    polling_thread.raise_exceptions()
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 111, in raise_exceptions
    raise self.exception_info
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 93, in run
    task(*args, **kwargs)
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 688, in __retrieve_updates
    updates = self.get_updates(offset=(self.last_update_id + 1),
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 660, in get_updates
    json_updates = apihelper.get_updates(
                   ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 332, in get_updates
    return _make_request(token, method_url, params=payload)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 168, in _make_request
    json_result = _check_result(method_name, result)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 195, in _check_result
    raise ApiTelegramException(method_name, result, result_json)
telebot.apihelper.ApiTelegramException: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running

2025-02-03 11:03:28,955 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=1&timeout=20 HTTP/1.1" 409 143
2025-02-03 11:03:28,957 - ERROR - Threaded polling exception: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
2025-02-03 11:03:28,958 - ERROR - Exception traceback:
Traceback (most recent call last):
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 1234, in __threaded_polling
    polling_thread.raise_exceptions()
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 111, in raise_exceptions
    raise self.exception_info
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 93, in run
    task(*args, **kwargs)
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 688, in __retrieve_updates
    updates = self.get_updates(offset=(self.last_update_id + 1),
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 660, in get_updates
    json_updates = apihelper.get_updates(
                   ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 332, in get_updates
    return _make_request(token, method_url, params=payload)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 168, in _make_request
    json_result = _check_result(method_name, result)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 195, in _check_result
    raise ApiTelegramException(method_name, result, result_json)
telebot.apihelper.ApiTelegramException: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running

2025-02-03 11:03:33,446 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=1&timeout=20 HTTP/1.1" 409 143
2025-02-03 11:03:33,461 - ERROR - Threaded polling exception: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
2025-02-03 11:03:33,462 - ERROR - Exception traceback:
Traceback (most recent call last):
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 1234, in __threaded_polling
    polling_thread.raise_exceptions()
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 111, in raise_exceptions
    raise self.exception_info
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 93, in run
    task(*args, **kwargs)
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 688, in __retrieve_updates
    updates = self.get_updates(offset=(self.last_update_id + 1),
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 660, in get_updates
    json_updates = apihelper.get_updates(
                   ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 332, in get_updates
    return _make_request(token, method_url, params=payload)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 168, in _make_request
    json_result = _check_result(method_name, result)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 195, in _check_result
    raise ApiTelegramException(method_name, result, result_json)
telebot.apihelper.ApiTelegramException: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running

2025-02-03 11:03:38,578 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=1&timeout=20 HTTP/1.1" 409 143
2025-02-03 11:03:38,579 - ERROR - Threaded polling exception: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
2025-02-03 11:03:38,580 - ERROR - Exception traceback:
Traceback (most recent call last):
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 1234, in __threaded_polling
    polling_thread.raise_exceptions()
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 111, in raise_exceptions
    raise self.exception_info
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 93, in run
    task(*args, **kwargs)
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 688, in __retrieve_updates
    updates = self.get_updates(offset=(self.last_update_id + 1),
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 660, in get_updates
    json_updates = apihelper.get_updates(
                   ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 332, in get_updates
    return _make_request(token, method_url, params=payload)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 168, in _make_request
    json_result = _check_result(method_name, result)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 195, in _check_result
    raise ApiTelegramException(method_name, result, result_json)
telebot.apihelper.ApiTelegramException: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running

2025-02-03 11:03:45,237 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=1&timeout=20 HTTP/1.1" 409 143
2025-02-03 11:03:45,239 - ERROR - Threaded polling exception: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
2025-02-03 11:03:45,240 - ERROR - Exception traceback:
Traceback (most recent call last):
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 1234, in __threaded_polling
    polling_thread.raise_exceptions()
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 111, in raise_exceptions
    raise self.exception_info
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 93, in run
    task(*args, **kwargs)
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 688, in __retrieve_updates
    updates = self.get_updates(offset=(self.last_update_id + 1),
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 660, in get_updates
    json_updates = apihelper.get_updates(
                   ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 332, in get_updates
    return _make_request(token, method_url, params=payload)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 168, in _make_request
    json_result = _check_result(method_name, result)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 195, in _check_result
    raise ApiTelegramException(method_name, result, result_json)
telebot.apihelper.ApiTelegramException: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running

2025-02-03 11:03:54,761 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=1&timeout=20 HTTP/1.1" 409 143
2025-02-03 11:03:54,762 - ERROR - Threaded polling exception: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
2025-02-03 11:03:54,763 - ERROR - Exception traceback:
Traceback (most recent call last):
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 1234, in __threaded_polling
    polling_thread.raise_exceptions()
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 111, in raise_exceptions
    raise self.exception_info
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 93, in run
    task(*args, **kwargs)
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 688, in __retrieve_updates
    updates = self.get_updates(offset=(self.last_update_id + 1),
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 660, in get_updates
    json_updates = apihelper.get_updates(
                   ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 332, in get_updates
    return _make_request(token, method_url, params=payload)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 168, in _make_request
    json_result = _check_result(method_name, result)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 195, in _check_result
    raise ApiTelegramException(method_name, result, result_json)
telebot.apihelper.ApiTelegramException: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running

2025-02-03 11:04:07,360 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=1&timeout=20 HTTP/1.1" 409 143
2025-02-03 11:04:07,365 - ERROR - Threaded polling exception: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
2025-02-03 11:04:07,367 - ERROR - Exception traceback:
Traceback (most recent call last):
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 1234, in __threaded_polling
    polling_thread.raise_exceptions()
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 111, in raise_exceptions
    raise self.exception_info
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 93, in run
    task(*args, **kwargs)
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 688, in __retrieve_updates
    updates = self.get_updates(offset=(self.last_update_id + 1),
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 660, in get_updates
    json_updates = apihelper.get_updates(
                   ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 332, in get_updates
    return _make_request(token, method_url, params=payload)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 168, in _make_request
    json_result = _check_result(method_name, result)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 195, in _check_result
    raise ApiTelegramException(method_name, result, result_json)
telebot.apihelper.ApiTelegramException: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running

2025-02-03 11:04:13,659 - DEBUG - Starting new HTTPS connection (1): api.telegram.org:443
2025-02-03 11:04:15,039 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getMe HTTP/1.1" 200 269
2025-02-03 11:04:15,040 - INFO - Bot ulandi - @QuizEnglishwords_bot
2025-02-03 11:04:15,462 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getMe HTTP/1.1" 200 269
2025-02-03 11:04:15,464 - DEBUG - Starting new HTTPS connection (1): api.telegram.org:443
2025-02-03 11:04:20,881 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=1&timeout=20 HTTP/1.1" 409 143
2025-02-03 11:04:20,886 - ERROR - Threaded polling exception: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
2025-02-03 11:04:20,889 - ERROR - Exception traceback:
Traceback (most recent call last):
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 1234, in __threaded_polling
    polling_thread.raise_exceptions()
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 111, in raise_exceptions
    raise self.exception_info
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 93, in run
    task(*args, **kwargs)
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 688, in __retrieve_updates
    updates = self.get_updates(offset=(self.last_update_id + 1),
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 660, in get_updates
    json_updates = apihelper.get_updates(
                   ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 332, in get_updates
    return _make_request(token, method_url, params=payload)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 168, in _make_request
    json_result = _check_result(method_name, result)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 195, in _check_result
    raise ApiTelegramException(method_name, result, result_json)
telebot.apihelper.ApiTelegramException: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running

2025-02-03 11:04:25,189 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=1&timeout=20 HTTP/1.1" 409 143
2025-02-03 11:04:25,191 - ERROR - Threaded polling exception: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
2025-02-03 11:04:25,192 - ERROR - Exception traceback:
Traceback (most recent call last):
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 1234, in __threaded_polling
    polling_thread.raise_exceptions()
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 111, in raise_exceptions
    raise self.exception_info
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 93, in run
    task(*args, **kwargs)
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 688, in __retrieve_updates
    updates = self.get_updates(offset=(self.last_update_id + 1),
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 660, in get_updates
    json_updates = apihelper.get_updates(
                   ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 332, in get_updates
    return _make_request(token, method_url, params=payload)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 168, in _make_request
    json_result = _check_result(method_name, result)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 195, in _check_result
    raise ApiTelegramException(method_name, result, result_json)
telebot.apihelper.ApiTelegramException: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running

2025-02-03 11:04:29,780 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=1&timeout=20 HTTP/1.1" 409 143
2025-02-03 11:04:29,783 - ERROR - Threaded polling exception: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
2025-02-03 11:04:29,784 - ERROR - Exception traceback:
Traceback (most recent call last):
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 1234, in __threaded_polling
    polling_thread.raise_exceptions()
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 111, in raise_exceptions
    raise self.exception_info
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 93, in run
    task(*args, **kwargs)
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 688, in __retrieve_updates
    updates = self.get_updates(offset=(self.last_update_id + 1),
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 660, in get_updates
    json_updates = apihelper.get_updates(
                   ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 332, in get_updates
    return _make_request(token, method_url, params=payload)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 168, in _make_request
    json_result = _check_result(method_name, result)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 195, in _check_result
    raise ApiTelegramException(method_name, result, result_json)
telebot.apihelper.ApiTelegramException: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running

2025-02-03 11:04:34,386 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=1&timeout=20 HTTP/1.1" 409 143
2025-02-03 11:04:34,389 - ERROR - Threaded polling exception: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
2025-02-03 11:04:34,395 - ERROR - Exception traceback:
Traceback (most recent call last):
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 1234, in __threaded_polling
    polling_thread.raise_exceptions()
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 111, in raise_exceptions
    raise self.exception_info
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 93, in run
    task(*args, **kwargs)
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 688, in __retrieve_updates
    updates = self.get_updates(offset=(self.last_update_id + 1),
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 660, in get_updates
    json_updates = apihelper.get_updates(
                   ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 332, in get_updates
    return _make_request(token, method_url, params=payload)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 168, in _make_request
    json_result = _check_result(method_name, result)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 195, in _check_result
    raise ApiTelegramException(method_name, result, result_json)
telebot.apihelper.ApiTelegramException: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running

2025-02-03 11:04:40,530 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=1&timeout=20 HTTP/1.1" 409 143
2025-02-03 11:04:40,531 - ERROR - Threaded polling exception: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
2025-02-03 11:04:40,533 - ERROR - Exception traceback:
Traceback (most recent call last):
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 1234, in __threaded_polling
    polling_thread.raise_exceptions()
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 111, in raise_exceptions
    raise self.exception_info
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 93, in run
    task(*args, **kwargs)
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 688, in __retrieve_updates
    updates = self.get_updates(offset=(self.last_update_id + 1),
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 660, in get_updates
    json_updates = apihelper.get_updates(
                   ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 332, in get_updates
    return _make_request(token, method_url, params=payload)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 168, in _make_request
    json_result = _check_result(method_name, result)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 195, in _check_result
    raise ApiTelegramException(method_name, result, result_json)
telebot.apihelper.ApiTelegramException: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running

2025-02-03 11:04:49,135 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=1&timeout=20 HTTP/1.1" 409 143
2025-02-03 11:04:49,136 - ERROR - Threaded polling exception: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
2025-02-03 11:04:49,137 - ERROR - Exception traceback:
Traceback (most recent call last):
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 1234, in __threaded_polling
    polling_thread.raise_exceptions()
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 111, in raise_exceptions
    raise self.exception_info
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 93, in run
    task(*args, **kwargs)
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 688, in __retrieve_updates
    updates = self.get_updates(offset=(self.last_update_id + 1),
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 660, in get_updates
    json_updates = apihelper.get_updates(
                   ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 332, in get_updates
    return _make_request(token, method_url, params=payload)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 168, in _make_request
    json_result = _check_result(method_name, result)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 195, in _check_result
    raise ApiTelegramException(method_name, result, result_json)
telebot.apihelper.ApiTelegramException: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running

2025-02-03 11:05:00,805 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=1&timeout=20 HTTP/1.1" 409 143
2025-02-03 11:05:00,807 - ERROR - Threaded polling exception: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
2025-02-03 11:05:00,808 - ERROR - Exception traceback:
Traceback (most recent call last):
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 1234, in __threaded_polling
    polling_thread.raise_exceptions()
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 111, in raise_exceptions
    raise self.exception_info
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 93, in run
    task(*args, **kwargs)
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 688, in __retrieve_updates
    updates = self.get_updates(offset=(self.last_update_id + 1),
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 660, in get_updates
    json_updates = apihelper.get_updates(
                   ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 332, in get_updates
    return _make_request(token, method_url, params=payload)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 168, in _make_request
    json_result = _check_result(method_name, result)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 195, in _check_result
    raise ApiTelegramException(method_name, result, result_json)
telebot.apihelper.ApiTelegramException: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running

2025-02-03 11:05:20,656 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=1&timeout=20 HTTP/1.1" 409 143
2025-02-03 11:05:20,658 - ERROR - Threaded polling exception: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
2025-02-03 11:05:20,659 - ERROR - Exception traceback:
Traceback (most recent call last):
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 1234, in __threaded_polling
    polling_thread.raise_exceptions()
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 111, in raise_exceptions
    raise self.exception_info
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 93, in run
    task(*args, **kwargs)
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 688, in __retrieve_updates
    updates = self.get_updates(offset=(self.last_update_id + 1),
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 660, in get_updates
    json_updates = apihelper.get_updates(
                   ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 332, in get_updates
    return _make_request(token, method_url, params=payload)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 168, in _make_request
    json_result = _check_result(method_name, result)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 195, in _check_result
    raise ApiTelegramException(method_name, result, result_json)
telebot.apihelper.ApiTelegramException: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running

2025-02-03 11:05:56,729 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=1&timeout=20 HTTP/1.1" 409 143
2025-02-03 11:05:56,730 - ERROR - Threaded polling exception: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
2025-02-03 11:05:56,731 - ERROR - Exception traceback:
Traceback (most recent call last):
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 1234, in __threaded_polling
    polling_thread.raise_exceptions()
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 111, in raise_exceptions
    raise self.exception_info
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 93, in run
    task(*args, **kwargs)
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 688, in __retrieve_updates
    updates = self.get_updates(offset=(self.last_update_id + 1),
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 660, in get_updates
    json_updates = apihelper.get_updates(
                   ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 332, in get_updates
    return _make_request(token, method_url, params=payload)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 168, in _make_request
    json_result = _check_result(method_name, result)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 195, in _check_result
    raise ApiTelegramException(method_name, result, result_json)
telebot.apihelper.ApiTelegramException: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running

2025-02-03 11:07:00,306 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=1&timeout=20 HTTP/1.1" 409 143
2025-02-03 11:07:00,339 - ERROR - Threaded polling exception: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
2025-02-03 11:07:00,342 - ERROR - Exception traceback:
Traceback (most recent call last):
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 1234, in __threaded_polling
    polling_thread.raise_exceptions()
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 111, in raise_exceptions
    raise self.exception_info
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 93, in run
    task(*args, **kwargs)
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 688, in __retrieve_updates
    updates = self.get_updates(offset=(self.last_update_id + 1),
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 660, in get_updates
    json_updates = apihelper.get_updates(
                   ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 332, in get_updates
    return _make_request(token, method_url, params=payload)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 168, in _make_request
    json_result = _check_result(method_name, result)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 195, in _check_result
    raise ApiTelegramException(method_name, result, result_json)
telebot.apihelper.ApiTelegramException: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running

2025-02-03 11:08:04,456 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=1&timeout=20 HTTP/1.1" 409 143
2025-02-03 11:08:04,457 - ERROR - Threaded polling exception: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
2025-02-03 11:08:04,466 - ERROR - Exception traceback:
Traceback (most recent call last):
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 1234, in __threaded_polling
    polling_thread.raise_exceptions()
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 111, in raise_exceptions
    raise self.exception_info
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 93, in run
    task(*args, **kwargs)
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 688, in __retrieve_updates
    updates = self.get_updates(offset=(self.last_update_id + 1),
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 660, in get_updates
    json_updates = apihelper.get_updates(
                   ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 332, in get_updates
    return _make_request(token, method_url, params=payload)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 168, in _make_request
    json_result = _check_result(method_name, result)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 195, in _check_result
    raise ApiTelegramException(method_name, result, result_json)
telebot.apihelper.ApiTelegramException: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running

2025-02-03 11:09:08,155 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=1&timeout=20 HTTP/1.1" 409 143
2025-02-03 11:09:08,165 - ERROR - Threaded polling exception: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
2025-02-03 11:09:08,166 - ERROR - Exception traceback:
Traceback (most recent call last):
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 1234, in __threaded_polling
    polling_thread.raise_exceptions()
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 111, in raise_exceptions
    raise self.exception_info
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 93, in run
    task(*args, **kwargs)
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 688, in __retrieve_updates
    updates = self.get_updates(offset=(self.last_update_id + 1),
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 660, in get_updates
    json_updates = apihelper.get_updates(
                   ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 332, in get_updates
    return _make_request(token, method_url, params=payload)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 168, in _make_request
    json_result = _check_result(method_name, result)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 195, in _check_result
    raise ApiTelegramException(method_name, result, result_json)
telebot.apihelper.ApiTelegramException: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running

2025-02-03 11:10:11,742 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=1&timeout=20 HTTP/1.1" 409 143
2025-02-03 11:10:11,744 - ERROR - Threaded polling exception: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
2025-02-03 11:10:11,746 - ERROR - Exception traceback:
Traceback (most recent call last):
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 1234, in __threaded_polling
    polling_thread.raise_exceptions()
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 111, in raise_exceptions
    raise self.exception_info
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 93, in run
    task(*args, **kwargs)
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 688, in __retrieve_updates
    updates = self.get_updates(offset=(self.last_update_id + 1),
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 660, in get_updates
    json_updates = apihelper.get_updates(
                   ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 332, in get_updates
    return _make_request(token, method_url, params=payload)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 168, in _make_request
    json_result = _check_result(method_name, result)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 195, in _check_result
    raise ApiTelegramException(method_name, result, result_json)
telebot.apihelper.ApiTelegramException: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running

2025-02-03 11:11:15,750 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=1&timeout=20 HTTP/1.1" 409 143
2025-02-03 11:11:15,752 - ERROR - Threaded polling exception: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
2025-02-03 11:11:15,763 - ERROR - Exception traceback:
Traceback (most recent call last):
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 1234, in __threaded_polling
    polling_thread.raise_exceptions()
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 111, in raise_exceptions
    raise self.exception_info
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 93, in run
    task(*args, **kwargs)
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 688, in __retrieve_updates
    updates = self.get_updates(offset=(self.last_update_id + 1),
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 660, in get_updates
    json_updates = apihelper.get_updates(
                   ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 332, in get_updates
    return _make_request(token, method_url, params=payload)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 168, in _make_request
    json_result = _check_result(method_name, result)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 195, in _check_result
    raise ApiTelegramException(method_name, result, result_json)
telebot.apihelper.ApiTelegramException: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running

2025-02-03 11:12:19,342 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=1&timeout=20 HTTP/1.1" 409 143
2025-02-03 11:12:19,343 - ERROR - Threaded polling exception: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
2025-02-03 11:12:19,352 - ERROR - Exception traceback:
Traceback (most recent call last):
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 1234, in __threaded_polling
    polling_thread.raise_exceptions()
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 111, in raise_exceptions
    raise self.exception_info
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 93, in run
    task(*args, **kwargs)
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 688, in __retrieve_updates
    updates = self.get_updates(offset=(self.last_update_id + 1),
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 660, in get_updates
    json_updates = apihelper.get_updates(
                   ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 332, in get_updates
    return _make_request(token, method_url, params=payload)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 168, in _make_request
    json_result = _check_result(method_name, result)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 195, in _check_result
    raise ApiTelegramException(method_name, result, result_json)
telebot.apihelper.ApiTelegramException: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running

2025-02-03 11:13:23,239 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=1&timeout=20 HTTP/1.1" 409 143
2025-02-03 11:13:23,240 - ERROR - Threaded polling exception: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
2025-02-03 11:13:23,241 - ERROR - Exception traceback:
Traceback (most recent call last):
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 1234, in __threaded_polling
    polling_thread.raise_exceptions()
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 111, in raise_exceptions
    raise self.exception_info
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 93, in run
    task(*args, **kwargs)
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 688, in __retrieve_updates
    updates = self.get_updates(offset=(self.last_update_id + 1),
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 660, in get_updates
    json_updates = apihelper.get_updates(
                   ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 332, in get_updates
    return _make_request(token, method_url, params=payload)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 168, in _make_request
    json_result = _check_result(method_name, result)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 195, in _check_result
    raise ApiTelegramException(method_name, result, result_json)
telebot.apihelper.ApiTelegramException: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running

2025-02-03 11:14:23,257 - DEBUG - Starting new HTTPS connection (1): api.telegram.org:443
2025-02-03 11:14:27,157 - DEBUG - https://api.telegram.org:443 "GET /bot8175153004:AAHtg3VJQzFeD26NceTrgKUCWfVp44iEJho/getUpdates?offset=1&timeout=20 HTTP/1.1" 409 143
2025-02-03 11:14:27,165 - ERROR - Threaded polling exception: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
2025-02-03 11:14:27,167 - ERROR - Exception traceback:
Traceback (most recent call last):
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 1234, in __threaded_polling
    polling_thread.raise_exceptions()
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 111, in raise_exceptions
    raise self.exception_info
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/util.py", line 93, in run
    task(*args, **kwargs)
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 688, in __retrieve_updates
    updates = self.get_updates(offset=(self.last_update_id + 1),
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/__init__.py", line 660, in get_updates
    json_updates = apihelper.get_updates(
                   ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 332, in get_updates
    return _make_request(token, method_url, params=payload)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 168, in _make_request
    json_result = _check_result(method_name, result)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gopher/go/myenv/lib/python3.12/site-packages/telebot/apihelper.py", line 195, in _check_result
    raise ApiTelegramException(method_name, result, result_json)
telebot.apihelper.ApiTelegramException: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running

